lib/utils/resolve-file-path.js

```diff
@@ -6,7 +6,11 @@ module.exports = function resolveFilePath(requireFile) {
     requireFile + '.js',
     requireFile + '.coffee',
     requireFile + '.js.coffee',
+    requireFile + '.coffee.js',
```

Comment:
I am not sure how much more work y'all need on this loader. 

Something to consider if there are more file types to support:

We could provide a list of file extensions or file patterns through the loader options. This makes it easier to use and expand the list of file extensions.